daml/ContingentClaims/Observable.daml

```diff
@@ -3,45 +3,41 @@
 -- SPDX-License-Identifier: Apache-2.0
 --
 
-{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE UndecidableInstances #-}
 
```

Comment:
Do we know if this pragma causes and issues, or is discouraged? In terms of (data-)dependencies or otherwise

Reply:
I'm not sure. But I've removed and everything seems to work :laughing: .